docs/content/releases/3.5.2.rst

```diff
@@ -0,0 +1,19 @@
+3.5.2
+=====
+
+Python dependency with conda:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Using conda which is the way we recommand installation it is now possible to use python 3.8.
+
+Relax version control on matplotlib:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Any version of matplotlib from 3.1.1 to 3.3.2 is now accepted.
```

Comment:
In the other PR you limited matplotlib to >=3.1.1 and <= 3.2.2 ?

Reply:
I made a mistake in the installation documentation, sorry. Thanks for the catch. I confirm 3.3.2.